src/utils.ts

```diff
@@ -35,7 +35,7 @@ export function walk(cwd: string, dir = cwd, files: string[] = []) {
 		}
 	});
 
-	return files;
+	return files.filter(v=>!v.includes('.DS_Store'));
```

Comment:
will this handle `.DS_Store` in nested directories?

it might be slightly more efficient to change the `else` in this method to `else if` and do the check there so that you don't have to do an extra iteration over the list of files

Reply:
yeah, I guess this happens during build time, so no need to prematurely optimize performance here. lgtm